<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBoardWidget</class>
 <widget class="QWidget" name="MainBoardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="lt_dbgControls">
     <item>
      <widget class="QPushButton" name="btn_Settings">
       <property name="text">
        <string>portSettings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_OpenPort">
       <property name="text">
        <string>openPort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ClosePort">
       <property name="text">
        <string>closePort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_MainBoard">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>BoardInfo</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lbl_BoardControl">
           <property name="text">
            <string>Board Controls</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Console" name="logsMainProcessor" native="true"/>
     </item>
     <item>
      <widget class="Console" name="logsCryptoProcessor" native="true"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Console</class>
   <extends>QWidget</extends>
   <header location="global">console.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
